### ISSUE TITLE
Map call fails due to lack of signed URL?

### ISSUE THREAD

**Issue by IndefiniteBen** [zoharby/plot_google_map#14]
See the below error ![image](https://user-images.githubusercontent.com/17594572/63595914-5e686680-c5ba-11e9-8a51-ca1177e8a8b2.png)
The problem is with the signature which needs to be used to sign the URL, [see docs here](https://developers.google.com/maps/documentation/maps-static/get-api-key#gen_sig). You can 'Allow unsigned usage' but I'm not a fan of doing that. I think fixing it requires changes to the code. I'll fork and make a pull request if I work it out.

**Comment by zoharby** [comment]
Makes sense. My key is still working although I haven't allowed unsigned usage. Maybe newly created projects will have a more strict behaviour. Feel free to add signing to the code and make a PR. 
I'm barely using MATLAB nowadays, so not really actively maintaining this project anymore...

**Comment by IndefiniteBen** [comment]
Yeah. I'm going to dump some links here for my own reference; I'll follow up later this year (could be next week).
* AFAIK current links are v2 https://developers.google.com/maps/documentation/maps-static/intro?hl=en_GB
* Premade apps for signing http://googlemaps.github.io/url-signing/index.html
* Read this entire page for most info https://developers.google.com/maps/documentation/maps-static/get-api-key#gen_sig
* Direct GCP link https://console.cloud.google.com/google/maps-apis/apis/static-maps-backend.googleapis.com/ (redirects to most recent project).
